src/sections/safe.tsx

```diff
@@ -1,196 +1,75 @@
-import { useEffect, useState } from 'react';
+import { useState } from 'react';
 
 //Stores
-import { useAppDispatch, useAppSelector } from '../store';
-import { safeActionsAsync, safeActions } from '../store/slices/safe';
+import { useAppSelector } from '../store';
 
 // HOPR Components
 import Section from '../future-hopr-lib-components/Section';
 import { useSigner } from '../hooks';
-import { utils } from 'ethers';
-function SafeSection() {
-  const dispatch = useAppDispatch();
-  const safe = useAppSelector((store) => store.safe);
-  const { account } = useAppSelector((store) => store.web3);
-  const { signer } = useSigner();
-  const [threshold, set_threshold] = useState(1);
-  const [owners, set_owners] = useState('');
 
-  useEffect(() => {
-    fetchInitialStateForSigner();
-  }, [signer]);
+import OwnersAndConfirmations from '../steps/safe/ownersAndConfirmations';
+import FundsToSafe from '../steps/safe/fundsToSafe';
+import SafeDeployed from '../steps/safe/safeDeployed';
 
-  const fetchInitialStateForSigner = async () => {
-    if (signer) {
-      dispatch(safeActions.resetState());
-      dispatch(safeActionsAsync.getSafesByOwnerThunk({ signer }));
-    }
-  };
+function SafeSection() {
+  const account = useAppSelector((store) => store.web3.account) as `0x${string}`;
+  const { signer } = useSigner();
+  const [step, set_step] = useState(2);
 
   if (!account) {
     return (
       <Section
         className="Section--safe"
         id="Section--safe"
         yellow
+        fullHeightMin
       >
         <h2>connect signer</h2>
       </Section>
     );
   }
 
-  return (
-    <Section
-      className="Section--safe"
-      id="Section--safe"
-      yellow
-    >
-      <h1>Safe</h1>
-      <h2>existing safes</h2>
-      {safe.safesByOwner.map((safeAddress) => (
-        <button
-          key={safeAddress}
-          onClick={() => {
-            if (signer) {
-              dispatch(
-                safeActionsAsync.getSafeInfoThunk({
-                  signer,
-                  safeAddress,
-                }),
-              );
-              dispatch(
-                safeActionsAsync.getAllSafeTransactionsThunk({
-                  signer,
-                  safeAddress,
-                }),
-              );
-              dispatch(
-                safeActionsAsync.getSafeDelegatesThunk({
```

Comment:
My changes removed this, where should this go?

Reply:
There where it was before would be best.
Your new subpage call Onboarding for now

Reply:
Created a new safe onboarding folder with the component, safe is now equal to develop branch